src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIRequestBuilder.java

```diff
@@ -28,9 +28,19 @@
  */
 public class ServiceNowTableAPIRequestBuilder extends RestAPIRequest.Builder {
   private static final String TABLE_API_URL_TEMPLATE = "%s/api/now/table/%s";
+  private static final String BATCH_API_URL_TEMPLATE = "%s/api/now/v1/batch";
+  private static final String SCHEMA_API_URL_TEMPLATE = "%s/api/now/doc/table/schema/%s";
```

Comment:
What's the difference between these 3 APIs, and when should each one be used?
  private static final String TABLE_API_URL_TEMPLATE = "%s/api/now/table/%s";
  private static final String BATCH_API_URL_TEMPLATE = "%s/api/now/v1/batch";
  private static final String SCHEMA_API_URL_TEMPLATE = "%s/api/now/doc/table/schema/%s";

Can you add some comments to clarify?

Reply:
Added comments in the code for all three URLs for better understanding. e8b33614f910fa16f06ffcf185ed0d3a23d9fda1